### PR TITLE
add to the insult script, add a magic eight ball script, add a spot me b...

### PR DESCRIPTION
**Pull request by huttneab**
...ro script

"tars spot me"

"tars call [name] a [insult]"

"tars will i get laid tonight"

### CODE REVIEW

**Comment by linqiu** [comment]
I feel like most of these make more sense as a reply rather than a hear  

Especially the `/will/` regex. Way to copy pasta!!

**Comment by huttneab** [comment]
i can squash if you want a nice commit history, if you care about that on this project?

File: scripts/magiceightball.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Description:
+#   ask me anything
+#
+# Notes:
```
Comment (linqiu) [inline scripts/magiceightball.coffee]: would you mind adding :point_down: so it'll show up when we ask for `tars help`

```
# Commands:
#   hubot will <text> - reply with 8 ball response
```

File: scripts/spotmebro.coffee [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Description:
+#   ask tars for a spot, bro
+#
+# Notes:
```
Comment (linqiu) [inline scripts/spotmebro.coffee]: and something appropriate here for commands

**Comment by linqiu** [comment]
nah. don't rewrite history :)

**Comment by huttneab** [comment]
It's not really rewriting history, just cleaning up code churn on a single piece of code, makes rebasing w/ master cleaner, else you will have to resolve merge conflicts for each commit during the rebase, which gets confusing and annoying. Not an issue for something this small, but sometimes people like best practices just to stay in practice.

**Pull request merged**